renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -509,6 +505,12 @@ __attribute__((visibility("default"))) EGLBoolean eglSwapBuffers(EGLDisplay dpy,
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_HEIGHT, &height);
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_WIDTH, &width);
 
+  GLInitParams &init = eglhooks.GetDriver()->GetInitParams();
+  int colorspace;
+  eglhooks.eglQuerySurface_real(dpy, surface, EGL_COLORSPACE, &colorspace);
+  // GL_SRGB8_ALPHA8 is specified as color-renderable, unlike GL_SRGB8.
+  init.isSRGB = init.colorBits == 32 && colorspace == EGL_COLORSPACE_sRGB;
```

Comment:
I think this is the correct check to determine isSRGB.

Not ideal to make GLInitParams public - I just realised I can probably do this calculation inside eglMakeCurrent.